xml/Microsoft.VisualBasic.ApplicationServices/NoStartupFormException.xml

```diff
@@ -24,7 +24,7 @@
  The Visual Basic Application Model requires that the application set the <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.MainForm%2A> property before running the <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.OnRun%2A> method. If the property is not set, the Visual Basic Application Model raises this exception. For more information, see [Extending the Visual Basic Application Model](~/docs/visual-basic/developing-apps/customizing-extending-my/extending-the-visual-basic-application-model.md).  
   
 > [!NOTE]
->  For information about how to set the startup form by using the **Project Designer**, see [Application Page, Project Designer (Visual Basic)](../Topic/Application%20Page,%20Project%20Designer%20\(Visual%20Basic\).md).  
+>  For information about how to set the startup form by using the **Project Designer**, see [Application Page, Project Designer (Visual Basic)](http://msdn.microsoft.com/library/8cec9fea-cd92-47ff-88dd-7c928f0b4a74).  
```

Comment:
The topic exists in docs: https://docs.microsoft.com/en-us/visualstudio/ide/reference/application-page-project-designer-visual-basic